host/cluster-down.yaml

```diff
@@ -257,6 +257,11 @@ spec:
         collectorName: "crictl-logs-haproxy-previous"
         command: "sh"
         args: ["-c", "crictl logs -p $(crictl ps -a --name haproxy -l --quiet) 2>&1"]
+    # sysctl parameters
+    - run:
+        collectorName: "sysctl-disable_ipv6"
+        command: "sh"
+        args: ["-c", "sysctl -a 2>/dev/null | grep disable_ipv6"]
```

Comment:
I'm curious is there any reason to limit this with the grep? Why not just get a list of all sysctl and allow analyzers to be written to check other things without limiting collection?